src/service/processing_selections.py

```diff
@@ -21,6 +21,10 @@
 
 MAX_SELECTION_IDS = 10000
 
+# The set descriptions go in the workspace metadata, which has a maximum size of 900 for the
```

Comment:
800?

Reply:
Workspace limit is 900 for len(key) + len(value), so max description is 900 - len(key). I just rounded down to 800